manubot/cite/tests/test_csl_dict.py

```diff
@@ -0,0 +1,70 @@
+import copy
+import pytest
+
+from manubot.cite.csl_dict import (
+    csl_item_set_standard_id)
```

Comment:
I think we should call this module `csl_item` rather than `csl_dict`, in line with the current terminology in functions. The "item" terminology is used [by the CSL docs](http://docs.citationstyles.org/en/stable/primer.html#item-metadata):

> Next up are the bibliographic details of the items you wish to cite: the item metadata.
> 
> For example, the bibliographic entry for a journal article may show the names of the authors, the year in which the article was published, the article title, the journal title, the volume and issue in which the article appeared, the page numbers of the article, and the article’s Digital Object Identifier (DOI). All these details help the reader identify and find the referenced work.
> 
> Reference managers make it easy to create a library of items. While many reference managers have their own way of storing item metadata, most support common bibliographic exchange formats such as BibTeX and RIS. The citeproc-js CSL processor introduced a JSON-based format for storing item metadata in a way citeproc-js could understand. Several other CSL processors have since adopted this “CSL JSON” format (also known as “citeproc JSON”).

The terminology we've adopted is `csl_data` for a list of `csl_item` dicts, and `csl_json` for `csl_data` that is JSON-serialized.

Note that Citation Styles Language contains other aspects that metadata storage. It's also a format for specifying a given reference style. Therefore, I could imagine `csl_dict` could be mistaken as somehow referring to a data structure related to the XML styles. Not that item is a very descriptive word, but at least it is what CSL docs seem to use.